sdk/cosmos/azure-cosmos/azure/cosmos/_cosmos_client_connection.py

```diff
@@ -1047,6 +1050,7 @@ def QueryItems(
         options: Optional[Mapping[str, Any]] = None,
         partition_key: Optional[PartitionKeyType] = None,
         response_hook: Optional[Callable[[Mapping[str, Any], Mapping[str, Any]], None]] = None,
+        request_context: Optional[Mapping[str, Any]] = None,
```

Comment:
Not sure why in some places this is typed as `Dict` and others `Mapping`?

Reply:
Removed request context as not needed